packages/core/lib/services/dids/index.test.js

```diff
@@ -99,6 +99,6 @@ describe('DID Service', () => {
 
   it('expect to register did dock', async () => {
```

Comment:
Can we include another test case for when something goes wrong.
Like, if no Keypair was found, or if there is insufficient for the transaction fee.